doc/Language/variables.rakudoc

```diff
@@ -1112,15 +1112,16 @@ short.
 
 =head2 Pre-defined lexical variables
 
-There are three special variables that are always available:
+There are four special variables that are always available:
```

Comment:
How should we word this? The special vars are not _always_ available.

Maybe something like

> There are four special variables that are available in certain scopes:

Reply:
I considered this, but you can access the variable anywhere, it's just Nil if we're not in the middle of a regex.

Reply:
But your point is taken. I'll update the text.